command/node_status.go

```diff
@@ -333,86 +343,109 @@ func (c *NodeStatusCommand) formatNode(client *api.Client, node *api.Node) int {
 	}
 
 	if c.short {
+		basic = append(basic, fmt.Sprintf("Host Volumes|%s", strings.Join(nodeVolumeNames(node), ",")))
 		basic = append(basic, fmt.Sprintf("Drivers|%s", strings.Join(nodeDrivers(node), ",")))
 		c.Ui.Output(c.Colorize().Color(formatKV(basic)))
-	} else {
-		// Get the host stats
-		hostStats, nodeStatsErr := client.Nodes().Stats(node.ID, nil)
-		if nodeStatsErr != nil {
-			c.Ui.Output("")
-			c.Ui.Error(fmt.Sprintf("error fetching node stats: %v", nodeStatsErr))
-		}
-		if hostStats != nil {
-			uptime := time.Duration(hostStats.Uptime * uint64(time.Second))
-			basic = append(basic, fmt.Sprintf("Uptime|%s", uptime.String()))
-		}
 
-		// Emit the driver info
-		if !c.verbose {
-			driverStatus := fmt.Sprintf("Driver Status| %s", c.outputTruncatedNodeDriverInfo(node))
-			basic = append(basic, driverStatus)
+		// Output alloc info
+		if err := c.outputAllocInfo(client, node); err != nil {
+			c.Ui.Error(fmt.Sprintf("Error querying node allocations: %s", err))
+			return 1
 		}
 
-		c.Ui.Output(c.Colorize().Color(formatKV(basic)))
+		return 0
```

Comment:
Terminating early here is very nice; it's much more obvious what the `short` flag displays here imo - thanks for doing this!